packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantBadges/RestaurantBadge.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <span
+        :class="[
+            [$style['c-restaurantBadge']], {
+                [$style['.c-restaurantBadge--isLarge']]: isLarge
+            }]"
```

Comment:
I don't think the first style needs the `[]` adding as it's part of the list. However to make it look consistent you could just revert to object syntax and pin the first style to `true` to make it visually easier to parse (IMO):
```suggestion
        :class="{
            [$style['c-restaurantBadge']]: true,
            [$style['.c-restaurantBadge--isLarge']]: isLarge
        }"
```

Reply:
just spotted a typo in my classname! Will do this locally and push!